addons/mail/static/src/components/follower_list_menu/follower_list_menu.xml

```diff
@@ -16,9 +16,6 @@
                                 Add Followers
                             </a>
                         </t>
-                        <a class="o_FollowerListMenu_addChannelsButton dropdown-item" href="#" role="menuitem" t-on-click="_onClickAddChannels">
-                            Add Channels
-                        </a>
```

Comment:
File `addons/mail/static/src/components/follower/follower.xml` could get a little clean up too.

Well mostly
`<img class="o_Follower_avatar" t-attf-src="/web/image/{{ follower.resModel }}/{{ follower.resId }}/image_128" alt="Avatar"/>`
Can become something like
`<img class="o_Follower_avatar" t-att-src="follower.partner.avatarUrl" alt="Avatar"/>`

And partner `partner.avatarUrl` reference has to be used in `useStore` in the corresponding js file for proper update on change.

I'm just noting this here but feel free to tell me to make a JS pass if you prefer.

Reply:
Seems unrelated change compare to this PR purpose, isn't it ?

Reply:
Ah ok I understand ... indeed it was some "generic avatar computation".